grammars/less.cson

```diff
@@ -456,3 +455,244 @@
+      {
+        'include': '#property_values'
+      }
+      {
+        'include': '#variables'
+      }
+    ]
+  'function_default':
+    'match': '\\b(default)(\\()(\\))'
+    'captures':
+      '1':
+        'name': 'support.function.default.less'
+      '2':
+        'name': 'meta.brace.round.css'
+      '3':
+        'name': 'meta.brace.round.css'
+  'function_unit_checking':
+    'begin': '\\b(isunit|ispixel|ispercentage|isem)(\\()'
+    'beginCaptures':
+      '1':
+        'name': 'support.function.unit-checking.less'
+      '2':
+        'name': 'meta.brace.round.css'
+    'end': '\\)'
+    'endCaptures':
+      '0':
+        'name': 'meta.brace.round.css'
+    'patterns': [
+      {
+        'include': '#property_values'
+      }
+      {
+        'include': '#variables'
+      }
+    ]
+  'mixins':
+    'name': 'meta.mixin.less'
+    'begin': '((\\.)[_a-zA-Z][a-zA-Z0-9_-]*)(?=\\s*\\()'
+    'beginCaptures':
+      '0':
+        'name': 'meta.definition.mixin.less'
+      '1':
+        'name': 'entity.name.mixin.less'
+      '2':
+        'name': 'punctuation.definition.mixin.less'
+    'end': '(?<=})'
+    'patterns': [
+      {
+        'include': '#mixin_parameters'
+      }
+      {
+        'include': '#mixin_guards'
+      }
+      {
+        'include': '$self'
+      }
+    ]
+  'mixin_parameters':
+    'name': 'meta.parameters.less'
+    'begin': '\\((?=\\s*@|\\))'
+    'beginCaptures':
+      '0':
+        'name': 'punctuation.definition.parameters.begin.bracket.round.less'
+    'end': '\\)'
+    'endCaptures':
+      '0':
+        'name': 'punctuation.definition.parameters.end.bracket.round.less'
+    'patterns': [
+      {
+        'include': '#mixin_parameter_semicolon'
+      }
+      {
+        'include': '#mixin_parameter_comma'
+      }
+      {
+        'include': '#mixin_parameter'
+      }
+    ]
+  'mixin_parameter_semicolon':
+    'begin': '(@)[a-zA-Z][a-zA-Z0-9-]*(?=.*?;)'
+    'beginCaptures':
+      '0':
+        'name': 'variable.parameter.less'
+      '1':
+        'name': 'punctuation.definition.variable.less'
+    'end': ';'
+    'endCaptures':
+      '0':
+        'name': 'punctuation.separator.parameter.less'
+    'patterns': [
+      {
+        'match': ':'
+        'name': 'punctuation.separator.key-value.less'
+      }
+      {
+        'include': 'source.css#numeric-values'
+      }
+      {
+        'include': 'source.css#color-keywords'
+      }
+      {
+        'include': 'source.css#functions'
+      }
+      {
+        'include': '#less_builtin_functions'
+      }
+      {
+        'include': 'source.css#property-keywords'
+      }
+      {
+        'include': '#strings'
+      }
+      {
+        'include': 'source.css#commas'
+      }
+    ]
+  'mixin_parameter_comma':
+    'begin': '(@)[a-zA-Z][a-zA-Z0-9-]*(?=.*?,\\s*@)'
+    'beginCaptures':
+      '0':
+        'name': 'variable.parameter.less'
+      '1':
+        'name': 'punctuation.definition.variable.less'
+    'end': '(,)(?=\\s*@)'
```

Comment:
The `,` doesn't need to have capture parentheses around it because the rest of the match is a lookahead.  You can safely target the 0th capture.

Reply:
Should be good to go now.